lib/manageiq/session_store.rb

```diff
@@ -0,0 +1,88 @@
+module ManageIQ
+  module SessionStore
+    def self.store
+      @store
+    end
+
+    def self.store=(session_store)
+      adapter = case session_store
+                when :active_record_store then ActiveRecordStoreAdapter.new
+                when :memory_store        then MemoryStoreAdapter.new
+                when :mem_cache_store     then MemCacheStoreAdapter.new
+                else
+                  raise "Please provide a valid Session Store"
+                end
```

Comment:
Can this be simplified to

```ruby
adapter = "#{session_store}_adapter".camelize.safe_constantize.try(:new)
raise "Please provide a valid Session Store" unless adapter
```

Reply:
Probably.  I assume by asking "can" you would prefer that form over the `case`, correct?

Reply:
It's slimmer and doesn't couple the Adapter list to the class, which is why I think it's a little cleaner, but I'm fine either way.

Reply:
> ... and doesn't couple the Adapter list to the class...

This is what sold me, and that is a really good point.  Will change.  👍